sktime/markov/_base.py

```diff
@@ -266,19 +270,22 @@ def score_cv(estimator: _MSMBaseEstimator, dtrajs, lagtime, n=10, count_mode="sl
         dynamics simulation. J. Chem. Theory Comput. 11, 5002-5011 (2015).
 
     """
-    from sktime.markov import TransitionCountEstimator
     from sktime.util import ensure_dtraj_list
     dtrajs = ensure_dtraj_list(dtrajs)  # ensure format
     if count_mode not in ('sliding', 'sample'):
         raise ValueError('score_cv currently only supports count modes "sliding" and "sample"')
     sliding = count_mode == 'sliding'
     scores = []
     for fold in range(n):
-        dtrajs_split = blocksplit_dtrajs(dtrajs, lag=lagtime, sliding=sliding, random_state=random_state)
+        if blocksplit:
+            dtrajs_split = blocksplit_dtrajs(dtrajs, lag=lagtime, sliding=sliding, random_state=random_state)
+        else:
+            dtrajs_split = dtrajs
+            if len(dtrajs_split) <= 1:
+                raise ValueError("Need at least two trajectories if blocksplit is not used to decompose the data.")
         dtrajs_train, dtrajs_test = cvsplit_dtrajs(dtrajs_split, random_state=random_state)
 
-        cc = TransitionCountEstimator(lagtime, count_mode).fit(dtrajs_train).fetch_model().submodel_largest()
-        model = estimator.fit(cc).fetch_model()
+        model = fit_fetch(dtrajs_train)
```

Comment:
So `fit_fetch` here enables a set of methods including but not limited to the ones deleted by this PR (i.e., 

```
cc = TransitionCountEstimator(lagtime, count_mode).fit(dtrajs_train).fetch_model().submodel_largest()
model = estimator.fit(cc).fetch_model()
```

?

I like the functionality, but it makes the code a lot less readable here

Reply:
the code becomes more general by this. the issue is that oom msms are estimated by dtrajs (because of this double counting business, i am not completely happy with the way things are right now) and ml-msms are estimated with a count matrix. so if `estimator` is an oom msm estimator, the previous code cannot run. I thought `fit_fetch` to be understandable in the sense of eg `fit_transform`.

Reply:
yes, it is understandable in that sense, I agree! it's just missing the words `TransitionCountEstimator` e.g. makes the code harder to read because it's not obvious what kind of thing is fitting/fetching. does that make sense

Reply:
makes sense :) the point is that for instance the oom msm doesnt really even use the transition count estimator because it does its own counting / bootstrapping. I guess I can add a comment to the code to make it easier to follow

Reply:
makes sense yeah i think i am more advocating for a comment than a change in any code. or like a really explicit examples doc - which can be done later. happy to merge w/out this, but would want to see it eventually